tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -338,13 +338,16 @@ private void CancelButtonPressed()
     /// Switches the repos shown to the account selected.
     /// </summary>
     [RelayCommand]
-    private async Task MenuItemClick(string selectedItemName)
+    private void MenuItemClick(string selectedItemName)

```

Comment:
Not very clear why this change fixes the UI. Async/await should help the UI be more responsive.

Reply:
I believe it is because I'm calling `await MenuItemClick` from the ViewModel and somehow the code execution pauses at `GetRepositoriesAsync`  I do have a bug on me to move this to the code-behind thought since this is more UI logic, than ViewModel logic.